.travis.yml

```diff
@@ -1,31 +1,39 @@
+sudo: false
 language: rust
+rust: nightly
+
+# Cache cargo symbols for faster build
+cache: cargo
 
+# Dependencies of kcov, used by coverage
 addons:
   apt:
     packages:
       - libcurl4-openssl-dev
       - libelf-dev
       - libdw-dev
+      - binutils-dev
+      - cmake # also required for cargo-update
+    sources:
+      - kalakris-cmake
 
-rust:
-  - nightly
+# Who needs exports when you can have travis do the work?
+env:
+  - PATH=$HOME/.cargo/bin:$PATH CARGO_FEATURES_DEFAULT=["nightly"]
```

Comment:
``CARGO_FEATURES_DEFAULT=["nightly"]`` This is set to retain overriding of feature flags if you'd rather this just compile without the nightly flag (it doesn't seem like its really being used) I can remove this and the feature in Cargo.toml.